dev/com.ibm.ws.org.apache.cxf.cxf.rt.databinding.jaxb.3.2/src/org/apache/cxf/jaxb/JAXBDataBinding.java

```diff
@@ -392,29 +411,54 @@ public synchronized void initialize(Service service) {
 
             if (col.getXmlSchemas().length > 1) {
                 // someone has already filled in the types
+                if (LOG.isLoggable(Level.FINE)) { // Liberty change begin
+                   LOG.fine("initialize: Got schemas for service: " + (serviceInfo != null ? serviceInfo.getName() : "null"));
```

Comment:
yes, the ServiceInfo class does not have toString() method, so just logging serviceInfo will not be useful as it will only print stl "initialize: Got schemas for service:  org.apache.cxf.service.model.ServiceInfo@20a4a2d2"